### ISSUE TITLE
TypeError: Cannot read property 'length' of undefined when using FormData

### ISSUE THREAD

**Issue by te-je** [bitinn/node-fetch#169]
I'm trying to follow the FormData example from the readme. Here's my code:

```
let form = new FormData();
form.append("username", "username");
form.append("_password", "password");
return fetch(url.format(urlParts), { method: "POST",  body: form });
```

But I get this error:
```
TypeError: Cannot read property 'length' of undefined
    at C:\Users\...\node_modules\node-fetch\index.js:97:113
    at new Fetch (C:\Users\...\node_modules\node-fetch\index.js:49:9)
    at Fetch (C:\Users\...\node_modules\node-fetch\index.js:37:10)
    at Promise.resolve.then.then.then (C:\Users\...\migrate.js:78:20)
    at process._tickCallback (internal/process/next_tick.js:103:7)
```

**Comment by yatsyk** [comment]
please check #167 
previous version of form-data should be ok. you can install it with command:
`npm i form-data@1.0.1 --save`

**Comment by bitinn** [comment]
Fixed in `node-fetch 1.6.3`, note that you need `form-data 2.1.0` as well.
